extensions/Worker.Extensions.ServiceBus/src/ServiceBusTriggerAttribute.cs

```diff
@@ -73,6 +73,7 @@ public ServiceBusTriggerAttribute(string topicName, string subscriptionName)
         /// <summary>
         /// Gets or sets the configuration to enable batch processing of events. Default value is "false".
         /// </summary>
+        [DefaultValue(false)]
```

Comment:
Since line 10 has `private bool _isBatched = false;` we know to set the DefaultValue to false.

Reply:
Will you be also releasing this package? or are you aware of any plans for that?

Reply:
Unless there is other work we are waiting on for this extension, then yes, I will bump the version in another PR and release it.